src/kibana/components/doc_table/components/table_row.js

```diff
@@ -104,10 +105,10 @@ define(function (require) {
           }
 
           $scope.columns.forEach(function (column) {
-            newHtmls.push(cellTemplate({
+            newHtmls.push(noWhiteSpace(cellTemplate({
```

Comment:
You shouldn't need the `noWhiteSpace` filter here. The `cellTemplate` was already stripped of whitespace so any that shows up in the results is from the value and should be shown.